examples/hacker_news_assets/hacker_news_assets/resources/__init__.py

```diff
@@ -78,9 +82,11 @@
 
 RESOURCES_LOCAL = {
     "parquet_io_manager": local_partitioned_parquet_io_manager,
-    "warehouse_io_manager": local_partitioned_parquet_io_manager,
+    "warehouse_io_manager": duckdb_partitioned_parquet_io_manager.configured(
+        {"duckdb_path": "hn_duckdb.duckdb"}
+    ),
     "pyspark": configured_pyspark,
     "warehouse_loader": snowflake_io_manager_prod,
```

Comment:
this cannot work. Snowflake is not available locally

Reply:
I think it is only used by the old non-asset-based implementation. Is this assumption correct? If yes, it could simply be deleted.

Reply:
That's right - planning to just delete it.